tests/unit/plugins/module_utils/test_mysql_user.py

```diff
@@ -9,7 +9,8 @@
     handle_grant_on_col,
     has_grant_on_col,
     normalize_col_grants,
-    sort_column_order
+    sort_column_order,
+    privileges_unpack
```

Comment:
```suggestion
    privileges_unpack,
```
it's a general recommendation I saw in several Python books